### DISCUSSION TITLE
How to group errors for dashboard

### DISCUSSION THREAD

**Discussion by Sentry7** [getsentry/support#54] [Support] [answered]
I'm trying to setup a dashboard where I can group different 400, 500 errors together. I dont see any option included multiple values to a single query i.e like error_code: ("400", "401", "500") instead I can only map 1 tag to 1 value. 
Any syntax to be used? or other alternatives please.

**Comment by danielkhan** [comment, marked as answer]
@Sentry7, here are the [docs](https://docs.sentry.io/product/sentry-basics/search/#multiple-values-on-the-same-key) for grouping multiple values on the same key. 

In your example, square brackets should do the trick: `error_code:["400", "401", "500"]`.
